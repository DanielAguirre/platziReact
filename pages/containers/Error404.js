import React from 'react';
import { Link } from 'react-router-dom';

const Error404 = () => (
  <section name="Home">
    <h1>Error 404</h1>
    <Link to="/">Go back to Home</Link>
  </section>
);

export default Error404;
