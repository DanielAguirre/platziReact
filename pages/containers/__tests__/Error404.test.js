import Error404 from '../Error404';

it('Render the Error404 componet', () => {
  const componet = shallow(<Error404/>);
  expect(componet).toMatchSnapshot();
});