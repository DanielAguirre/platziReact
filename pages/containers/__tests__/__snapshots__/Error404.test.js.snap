// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render the Error404 componet 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <section
    name="Home"
>
    <h1>
        Error 404
    </h1>
    <Link
        replace={false}
        to="/"
    >
        Go back to Home
    </Link>
</section>,
  "nodes": Array [
    <section
      name="Home"
>
      <h1>
            Error 404
      </h1>
      <Link
            replace={false}
            to="/"
      >
            Go back to Home
      </Link>
</section>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <Error404 />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": Error404 {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <section
          name="Home"
>
          <h1>
                    Error 404
          </h1>
          <Link
                    replace={false}
                    to="/"
          >
                    Go back to Home
          </Link>
</section>,
        "_debugID": 2,
        "_renderedOutput": <section
          name="Home"
>
          <h1>
                    Error 404
          </h1>
          <Link
                    replace={false}
                    to="/"
          >
                    Go back to Home
          </Link>
</section>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Error404 />,
}
`;
